fars/booking/static/js/month.js

```diff
@@ -43,20 +43,20 @@ $(document).ready(function() {
                 start: $(this).attr('start'),
                 end: $(this).attr('end'),
               };
+              var today = moment();
+              var classNames = [];
+              if (moment(event.end)._d.getTime() < today._d.getTime()) {

```

Comment:
not sure, but is `_d.getTime()` really necessary? I think `momentA < momentB` works fine?